### ISSUE TITLE
Narrator/Screen Reader is incorrectly reading the Draggable ID instead of Draggable item Label.

### ISSUE THREAD

**Issue by AmitChopade1** [atlassian/react-beautiful-dnd#2445]
### Expected behavior

1. Screen Reader should announce the Menu items names from menu list while performing drag and drop functionality, between menu items

### Actual behavior

1. Screen reader is announcing incorrect and confusing names such as 'e170f8df-9e4f-f073-2544-a694d0a76015' which is Draggable-id provided by **react-beautiful-dnd** Library. while performing drag and drop functionality.

### Steps to reproduce

1. Add a draggable menu list, using the **react-beautiful-dnd** Library.
2. Add 4-5 menus to the Draggable list.
3. Open Narrator/Screen Reader, Turn off scan mode by **pressing insert+ spacebar**
4. Navigate to the pages. Press spacebar to select an item **(to perform Drag & Drop)** and then press up/down arrow to merge and observe the issue.
5. Screen Reader should announce the Menu items names while performing drag and drop functionality.

### What browser are you using?

MS Edge browser / Chrome
